kolibri_instant_schools_plugin/assets/src/vue/sign-up-page/index.vue

```diff
@@ -122,6 +122,7 @@
       password: 'Password',
       enterPassword: 'Enter Password',
       confirmPassword: 'Confirm Password',
+      confirmPasswordPlaceholder: 'Enter Password Again',
```

Comment:
@jamalex @aronasorman @rtibbles Noted, will include it in one of the coming PR's